addons/hr_recruitment/models/hr_recruitment.py

```diff
@@ -63,6 +63,12 @@ class RecruitmentStage(models.Model):
     fold = fields.Boolean(
         "Folded in Recruitment Pipe",
         help="This stage is folded in the kanban view when there are no records in that stage to display.")
+    legend_blocked = fields.Char(
+        'Red Kanban Label', default=lambda s: _('Blocked'), translate=True, required=True)
```

Comment:
We have space enough to use real variable name, let us use self :) . This remark applies to all occurences of s.